### DISCUSSION TITLE
Lightweight Deflectometer (LWD)

### DISCUSSION THREAD

**Discussion by m-sams-artc** [australiangeomechanics/agsformat#41] [Q&A] [answered]
Does the current ags format (with or without AU localisation) support test results from lightweight deflectometers (LWD)? This is a test with growing popularity for in-situ testing of vertical modulus. Should be columns for time, force, displacement, dynamic modulus, vertical modulus, or others as deemed suitable.

**Comment by SolidRockProgrammer** [comment, marked as answer]
This is on the list for AGS 4.2 UK standard version, when there is a draft developed you can start to use it on your projects.
